commands/command_prune.go

```diff
@@ -232,7 +242,7 @@ func pruneCheckErrors(taskErrors []error) {
 	}
 }
 
-func pruneTaskDisplayProgress(progressChan PruneProgressChan, waitg *sync.WaitGroup, spinner *progress.Spinner) {
+func pruneTaskDisplayProgress(progressChan PruneProgressChan, waitg *sync.WaitGroup, task *tasklog.SimpleTask) {
```

Comment:
I'm not crazy about this function receiving a task, and then closing it. What do you think about sending it the `logger` instead, so that the task is created, enqueued, used, and completed all in this func?